src/Microsoft.AspNetCore.Blazor.Templates/content/Directory.Build.props

```diff
@@ -0,0 +1,4 @@
+<!-- Intentional does not import parent Directory.Build.props file to isolate templates projects from the rest of the repo -->
```

Comment:
nitpick: Intentional -> Intentionally

Here and in src/Microsoft.AspNetCore.Blazor.Templates/content/Directory.Build.targets

Reply:
Good catch. I fixed the grammar errors and incomplete code comments.